qutebrowser/keyinput/modeparsers.py

```diff
@@ -231,7 +228,7 @@ def execute(self, cmdstr, keytype, count=None):
         if keytype == self.Type.chain:
             hintmanager = objreg.get('hintmanager', scope='tab',
                                      window=self._win_id, tab='current')
-            hintmanager.fire(cmdstr)
+            hintmanager.handle_partial_key(cmdstr)
```

Comment:
This seems like another change coming from `master` which shouldn't be in this branch?